Makefile

```diff
@@ -14,7 +14,7 @@ all: environment install build validate auto-lint coverage
 environment:
 	@echo 🔧 ENVIRONMENT SETUP
 	make install-gmp
-	python -m pip install -U pip
+	python3 -m pip install -U pip
```

Comment:
Verified that the pip3 command is not needed but we might need some instructions on it in the setup.